vm/chef/cookbooks/erpnext/recipes/default.rb

```diff
@@ -36,41 +36,94 @@
   variables(frappeuser: node['erpnext']['frappe']['user'])
 end
 
-# Download installation script.
-remote_file "/home/#{node['erpnext']['frappe']['user']}/install.py" do
-  source node['erpnext']['install-script']
+bash 'Set MariaDB root password' do
+  code <<-EOH
+    mysqladmin -uroot password "#{node['erpnext']['mysql-root-password']}"
+
+    mysql -uroot -p"#{node['erpnext']['mysql-root-password']}" <<EOSQL
+      DELETE FROM mysql.user WHERE User='';
+      DELETE FROM mysql.user WHERE User='root' AND Host NOT IN ('localhost', '127.0.0.1', '::1');
+      FLUSH PRIVILEGES;
+EOSQL
+  EOH
+end
+
+# Add required MariaDB configuration
+cookbook_file '/etc/mysql/mariadb.conf.d/70-frappe.cnf' do
+  source 'frappe.cnf'
+  owner 'root'
+  group 'root'
+  mode 0664
+  action :create
+end
+
+service 'mysql' do
+  action :restart
+end
+
+# Install NodeJS
+remote_file "/home/#{node['erpnext']['frappe']['user']}/setup_nodejs" do
+  source 'https://deb.nodesource.com/setup_12.x'
```

Comment:
It's worth to add node major version to attributes.rb.

Reply:
Done